src/vignette.ts

```diff
@@ -22,7 +22,7 @@ interface ThumbnailOptions {
 }
 
 class Vignette {
-	private static imagePathRegExp: RegExp = /\/\/vignette\d?\.wikia/;
+	private static imagePathRegExp: RegExp = /\/\/vignette(\d|-poz)?\.wikia/;
```

Comment:
I've searched through chef-repo and there is nothing like that. SF uses `vignette.wikia-dev.com`.